target/linux/ramips/image/rt305x.mk

```diff
@@ -1183,6 +1183,16 @@ define Device/zorlik_zl5900v2
 endef
 TARGET_DEVICES += zorlik_zl5900v2
 
+define Device/zte_mf283plus
+  SOC := rt3352
+  IMAGE_SIZE := 15872k
+  DEVICE_VENDOR := ZTE
+  DEVICE_MODEL := MF283+
+  DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 kmod-usb-ledtrig-usbport \
+	kmod-usb-serial kmod-usb-serial-option kmod-usb-net-qmi-wwan uqmi
```

Comment:
I found no other devices using kmod-usb-net-qmi-wwan or uqmi in DEVICE_PACKAGES. Do you really need it?

Reply:
My statement is apparantly wrong, looks like I did a mistake during searching earlier.